boards/native/Makefile.features

```diff
@@ -1,6 +1,7 @@
 # Put defined MCU peripherals here (in alphabetical order)
 FEATURES_PROVIDED += periph_cpuid
 FEATURES_PROVIDED += periph_hwrng
+FEATURES_PROVIDED += periph_pm
```

Comment:
I would suggest moving this line to `cpu/native/Makefile.features`, as PM has not board specific configuration